Source/ASImageNode.mm

```diff
@@ -305,11 +305,11 @@ - (void)setPlaceholderColor:(UIColor *)placeholderColor
 
 - (NSObject *)drawParametersForAsyncLayer:(_ASDisplayLayer *)layer
 {
-  ASDN::MutexLocker l(__instanceLock__);
+  ASLockScopeSelf();
   
   ASImageNodeDrawParameters *drawParameters = [[ASImageNodeDrawParameters alloc] init];
   drawParameters->_image = [self _locked_Image];
-  drawParameters->_bounds = [self threadSafeBounds];
+  drawParameters->_bounds = self.bounds;
```

Comment:
Is this particular change necessary, and also safe? IIRC it was important to use threadSafeBounds here.

Reply:
Oh I didn't know. When I removed the FrameworkSubclasses import, the import for this field was lost and I figured it was OK to use bounds since this method is (allegedly) called on main. I put it back since the change is outside the purview of this diff 👍